modules/testserver/src/main/java/com/smartbear/readyapi4j/testserver/execution/ApiClientWrapper.java

```diff
@@ -135,34 +121,46 @@
             }
         } else if (response.getStatusInfo().getStatusCode() == 429) {
             throw new UsageLimitException(response.getStatusInfo().getStatusCode(), getResponseBody(response),
-                response.getHeaders());
+                    response.getHeaders());
         } else {
             throw new ApiException(response.getStatusInfo().getStatusCode(), getResponseBody(response),
-                response.getHeaders());
+                    response.getHeaders());
+        }
+    }
+
+    private String createQueryString(List<Pair> queryParams) {
+        StringBuilder b = new StringBuilder("?");
+        if (queryParams != null) {
+            for (Pair queryParam : queryParams) {
+                if (!queryParam.getName().isEmpty()) {
+                    b.append(escapeString(queryParam.getName()));
```

Comment:
I think doing all the appends in one statement, but on multiple lines, is less noisy.